ProcessMaker/pm-blocks
# FOUR-12763: Remove "ess" environments

This PR addresses a critical issue related to the automatic creation of environment variables, which was occurring excessively—surpassing the limit of 100 instances. The modifications implemented in this PR effectively resolve the problem, ensuring that the generation of environment variables adheres to the intended behavior without exceeding the specified threshold. The changes made are designed to enhance the stability and performance of the system by curbing the unintended proliferation of environment variables.

## Solution
- Remove "ess" environments form the Manifests

## How to Test

To validate the resolution of the issue related to excessive environment variable creation, follow these steps:

1. Confirm that the PM Blocks package is installed in your environment.
2. Adjust the `DEFAULT_PM_BLOCKS_BRANCH` from your .env to pull from this PR:
   ```
   DEFAULT_PM_BLOCKS_BRANCH=task/FOUR-12763
   ```
3. Execute the sync command as per your system's specifications.
`php artisan package-pm-blocks:sync-pm-blocks`
4. Verify that the sync process is successful and inspect both the database and the UI to ensure that there are no instances of excessive `ess` environment variables stored. 

## Related Tickets & Packages
[FOUR-12763](https://processmaker.atlassian.net/browse/FOUR-12763)


## Code Review Checklist




[FOUR-12763]: https://processmaker.atlassian.net/browse/FOUR-12763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
